cfgov/v1/jinja2/v1/sublanding-page/filterable.html

```diff
@@ -0,0 +1,9 @@
+{% extends "v1/sublanding-page/index.html" %}
+
+{% block content_main %}
+    {{ super() }}
```

Comment:
I wonder if there are parts of https://github.com/cfpb/consumerfinance.gov/blob/fc2e53621526410779b6a1a031f03609c562c5c7/cfgov/v1/jinja2/v1/sublanding-page/index.html#L21-L54 that would never be used in filterable pages, in which case maybe those pages should extend `v1/sublanding-page/index.html` as their own sub-templates.

Reply:
Potentially.. currently it's possible to add all kinds of content to filterable pages even though in practice I'm not sure which we do right now, or might want to use in future.

I think a preferable direction might be to see how we can get to a world where these page templates simply say something like 

```jinja
{% for block in page.content %}
    {% include_block block %}
{% endfor %}
```

or, maybe with our custom rendering that adds `block` classes:

```jinja
{% import 'v1/includes/templates/render_block.html' as render_block with context %}
{% for block in page.content %}
    {{ render_block.render(block, loop.index) }}
{% endfor %}
```

A lot of the noise in these page templates is due to this custom handling of different block types although really the page templates shouldn't need this kind of case-by-case logic.